xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -1228,6 +1261,10 @@ void CVideoPlayer::Prepare()
   m_CurrentAudio.lastdts = DVD_NOPTS_VALUE;
   m_CurrentVideo.lastdts = DVD_NOPTS_VALUE;
 
+  CJobManager::GetInstance().Submit([&]() {
+    m_callback.RequestVideoSettings(m_item);
```

Comment:
I think we addressed before the use of references in a JobManager lambda?

Reply:
thanks! fixed now